examples/bootstrap_server/bootstrap_server.ini

```diff
@@ -86,7 +86,8 @@ secret=4F7572536563726574
 [Endpoint]
 Delete=/0
 Delete=/1
-Server=2
+Server=1
+Discover=/
```

Comment:
This edit is done to trigger block2 transfer to bootstrap server "out of the box". Could/should be removed in final version

./build-bootstrap/bootstrap_server -f examples/bootstrap_server/bootstrap_server.ini -4
./build-client/lwm2mclient -4 -b -n bepa2